format/templates/markdown_table_resources.tmpl

```diff
@@ -8,15 +8,15 @@
     {{ else }}
         {{- indent 0 "#" }} Resources
 
-        | Name | Type |
-        |------|------|
+        | Name | Type | Comment |
+        |------|------|---------|
```

Comment:
This column should be optional and turned on by a flag.  Not everyone may want this and it would change existing behaviour.